solutions/beff/pm/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Game</title>
+        <link rel="stylesheet" href="style.css" />
+        <script src="script.js"></script>
+    </head>
+    <body>
+        <h1>Choose a game!</h1>
+        <h2>Click to run...</h2>
+        <!-- <div class="link">
+            <a href="#!" onclick="rockPaperScissors()"> rockPaperScissors</a>
+        </div> -->
+        <div class="link">
+            <a href="./rps/rps.html"> rockPaperScissors</a>
+        </div>
+        <div class="link">
+            <a href="#!" onclick="pokemonFight()"> pokemonFight</a>
```

Comment:
Using `onclick` in HTML (presentation layer) is ill-advised. 

Industry standard conforming to the [Unobtrusive_JavaScript](https://en.wikipedia.org/wiki/Unobtrusive_JavaScript) idea. 

This would mean connecting the event in your javascript file thus keeping the presentation layer separate from the logic layer.